jina/parsers/peapods/runtimes/head.py

```diff
@@ -0,0 +1,21 @@
+from jina.parsers.helper import add_arg_group
+
+
+def mixin_head_parser(parser):
+    """Mixing in arguments required by head peas and runtimes into the given parser.
+    :param parser: the parser instance to which we add arguments
+    """
+
+    gp = add_arg_group(parser, title='Head')
+
+    gp.add_argument(
```

Comment:
can u prepare the args to have multiple addresses? so that even if we add it later we have less breaking changes?

Reply:
why multiple addresses? `Uses_before`/`Uses_after` should be always single instances I think? At least with the current design that was what I intended. If you need to scale them, you need to scale the head.